aip/general/0162.md

```diff
@@ -342,14 +300,13 @@ ranges of time.
 Revisions are sometimes expensive to store, and there are valid use cases to
 want to remove one or more revisions from a resource's revision history.
 
-APIs **may** define a method to delete revisions, with a structure similar to
-`Delete` ([AIP-135][]) methods:
+APIs **may** define a standard `Delete` ([AIP-135][]) method to delete resource revisions
```

Comment:
This should be in a section that talks about revisions being a child collection and following the standard method AIPs (131,132,133,134,135).